src/test/java/service/AmountHandlerBalanceTest.java

```diff
@@ -0,0 +1,14 @@
+package service;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+public class AmountHandlerBalanceTest {
+
+    @Test
+    public void getAmount() {
+        AmountHandlerBalance amountHandlerBalance = new AmountHandlerBalance();
+        Assert.assertEquals("Balance value is wrong",Integer.valueOf(10),
```

Comment:
Integer.valueOf - what for?, the same for other similar cases

Reply:
Tested function return Integer type, but literal with expected value considered  as int.  Please advice.